### PR TITLE
add linter and babel, adjust code to follow linter rules

### PR DESCRIPTION
**Pull request by robinHurtado**
Adjust proptypes

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Introduce ESLint that aim to follow a style guide when writing code

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
EsLint will improve the way of writing code and avoid typos.

## Does this PR introduce a breaking change?
```
[ ] Yes
[ x] No
```

## Other information

### CODE REVIEW

**Comment by robinHurtado** [comment]
Hello @shmesa22, i think that is necessary to have a linter so i added in this PR.

File: src/FlexOptions.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import React from 'react';
+import { PropTypes } from 'prop-types';
```
Comment (alejandronanez) [inline src/FlexOptions.js]: Yay, thanks for adding this!

**Comment by robinHurtado** [comment]
@alejandronanez I Removed the `.babelrc` file and i updated my **yarn** to `1.10.1`

**Comment by alejandronanez** [comment]
@robinHurtado can you please run `yarn` one more time? The `integrity` field is still missing.

I opened an issue to solve this `yarn` _issue_ we're seeing here! https://github.com/react-medellin/flexbox-playground/issues/36

**Comment by robinHurtado** [comment]
Hi @alejandronanez the integrity field is back in yarn.lock, i will be investigating that issue and if i can solve it then i will be creating the PR for that issue.

**Pull request merged**